src/pages/Regions/RegionFollowers.tsx

```diff
@@ -0,0 +1,153 @@
+import React, {useEffect, useState} from 'react';
+import {useHistory, useParams} from 'react-router-dom';
+import {Avatar, Button, Card, Layout, Skeleton, Spin} from 'antd';
+import {CloseOutlined, PlusOutlined, RollbackOutlined} from '@ant-design/icons';
+import {
+  getRegionFollowers, 
+  getRegionAdministration, 
+  getRegionById, removeFollower, 
+} from "../../api/regionsApi";
+import CityDefaultLogo from "../../assets/images/default_city_image.jpg";
+import userApi from "../../api/UserApi";
+import "./Region.less";
+import RegionFollower from '../../models/Region/RegionFollower';
+import Title from 'antd/lib/typography/Title';
+import Spinner from '../Spinner/Spinner';
+import NotificationBoxApi from '../../api/NotificationBoxApi';
+import { Roles } from '../../models/Roles/Roles';
+
+const RegionFollowers = () => {
+    const {id} = useParams();
+    const history = useHistory();
+
+    const [loading, setLoading] = useState<boolean>(false);
+    const [photosLoading, setPhotosLoading] = useState<boolean>(false);
+    const [regionName, setRegionName] = useState<string>("");
+    const [followers, setFollowers] = useState<RegionFollower[]>([]);
+    const [activeUserRoles, setActiveUserRoles] = useState<string[]>([]);
+    const [isActiveUserRegionAdmin, setIsActiveUserRegionAdmin] = useState<boolean>(false);
+
+    const getFollowers = async () => {
+      setLoading(true);
+      const response = await getRegionById(id)
+      const response1 = await getRegionFollowers(id);
+      const response2 = await getRegionAdministration(id);

```

Comment:
Please, provide an appropriate name
This name doesn't meet with conventions

Reply:
Will fix it with the next commit